### PR TITLE
Quick start

### PR DESCRIPTION
**Pull request by rosemarymarano**
A Very Rough first draft of a Quick Start Guide. The steps/categories seem OK but we need to fill them in with the actual commands that users will perform. This guide is a tutorial for users to create a WebLogic Kubernetes deployment in a Kubernetes cluster. It must contain every detail needed to accomplish the goal with verification steps where needed. I'm putting it out there for the SMEs to fill in the details.

### CODE REVIEW

File: kubernetes/create-weblogic-domain-inputs.yaml [hunk 1]
```diff
@@ -8,13 +8,14 @@ version: create-weblogic-domain-inputs-v1
 adminPort: 7001
 
 # Name of the Admin Server
+# This name must not contain an underscope ("_"), and must be lowercase.
```
Comment (tbarnes-us) [inline kubernetes/create-weblogic-domain-inputs.yaml]: underscope --> underscore

File: kubernetes/create-weblogic-domain-inputs.yaml [hunk 2]
```diff
@@ -8,13 +8,14 @@ version: create-weblogic-domain-inputs-v1
...
+# This name must not contain an underscope ("_"), and must be lowercase. 
 adminServerName: admin-server
 
 # Name of the WebLogic domain to create
 domainName: base_domain
 
-# Unique id identifying a domain.
-# This id must be lowercase and unique across all domains in a Kubernetes cluster.
+# Unique ID identifying a domain. 
+# This ID must not contain an underscope ("_"), and must be lowercase and unique across all domains in a Kubernetes cluster.
```
Comment (tbarnes-us) [inline kubernetes/create-weblogic-domain-inputs.yaml]: underscope --> underscore

File: kubernetes/create-weblogic-domain-inputs.yaml [hunk 3]
```diff
@@ -26,6 +27,7 @@ clusterType: DYNAMIC
 startupControl: AUTO
 
 # Cluster name
+# This name must not contain an underscope ("_"), and must be lowercase.
```
Comment (tbarnes-us) [inline kubernetes/create-weblogic-domain-inputs.yaml]: underscope --> underscore

File: kubernetes/create-weblogic-domain-inputs.yaml [hunk 4]
```diff
@@ -35,6 +37,7 @@ configuredManagedServerCount: 2
 initialManagedServerReplicas: 2
 
 # Base string used to generate managed server names
+# This name must not contain an underscope ("_"), and must be lowercase.
```
Comment (tbarnes-us) [inline kubernetes/create-weblogic-domain-inputs.yaml]: underscope --> underscore

File: site/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+* Configure the operator to manage the domains in the domain namespace using `helm upgrade`
+* Configure the Traefik load balancer to manage the domains in the domain namespace.
+  * Use `helm upgrade` to configure the load balancer to monitor the Ingresses in the domain namespace
+* Create the domain home
+* Create the Docker image for the domain home in the image or use WebLogic binary image.
+  * Run WLST to create the domain in PV (remember to apply patch).
+
+## 6. Create a domain.
+* Edit the domain YAML file (can the defaults be used?)
+* Create a situational configuration template
```
Comment (tbarnes-us) [inline site/quickstart.md]: Optionally create a situational configuration template (for example, to override the domain home configuration of a database URL, username, and password).

Comment (tbarnes-us) [reply site/quickstart.md]: Also, this step should occur after the 'create domain home' step, but before the 'create domain resource' step.

**Comment by doxiao** [comment]
Hi Rosemary, is this quick start for Operator 2.0? 

The RP says it is to be merged into develop branch.  But there are changes in some of the files that have been removed in 2.0, for examples, the files that are related to the old create domain script in 1.x releases:  kubernetes/create-weblogic-domain-inputs.yaml , kubernetes/internal/create-weblogic-domain.sh and kubernetes/internal/domain-custom-resource-template.yaml. I am confused.

File: site/quickstart.md (hunk 5 above)
Comment (rosemarymarano) [reply site/quickstart.md]: I incorporated your comments/edits and you will see them in the next quickstart.md I push out.

**Pull request closed without merging**